src/components/TodoList/TodoList.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import TodoListItems from '../TodoListItems/TodoListItems';
+import TodoListHeader from '../TodoListHeader/TodoListHeader';
+import TodoInputText from '../TodoInputText/TodoInputText';
+
+const TodoList = () => {
+  // Get data from local storage
+  const getDataFromLocalStorage = () => JSON.parse(localStorage.getItem('react-todo')) || [];
+
+  const [todos, setTodos] = useState(getDataFromLocalStorage());
+
+  // Local Storage
+  useEffect(() => {
+    localStorage.setItem('react-todo', JSON.stringify(todos));
+  }, [todos]);
+
+  // Add new Todo
+  const handleClickAddTodo = (title) => {
+    const newTodo = { id: uuidv4(), title, completed: false };
+    setTodos((prevState) => [...prevState, newTodo]);
+  };
+
+  // Remove Todo
+  const handleRemoveTodo = (id) => {
+    setTodos(todos.filter((todo) => todo.id !== id));
+  };
```

Comment:
[Optional] 
- While using a set method of useState() it is recommended to use the previous State while setting the new state, in your case, kindly set the todos using the prevstate. 
```
setTodos((prevState)=>{prevState.filter((todo) => todo.id !== id))})
```